### PR TITLE
PROGRAMMES-5957 Adding related link tracking

### PR DESCRIPTION
**Pull request by votemike**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-5957

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by Tasiobg** [comment]
I think this may not work with external links, the javascript works different, you need to use the HTML property "data-extlinktrack" for external links.
This is an example https://github.com/bbc/programmes-frontend/commit/25a20acd08df39e1faa04d55099a3cf269a986bb

External links are the links pointing outside BBC

Don't need to use PHP for that, but If you think is better, I have no objections

File: src/DsAmen/Presenters/Domain/RelatedLink/related_link.html.twig [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <div class="related-links__item 1/1 1/2@gel3 1/3@gel3b 1/4@gel5">
-    <a class="gel-long-primer-bold" href="{{ related_link.getLink() }}">{{ related_link.getTitle() }}</a>
+    <a class="gel-long-primer-bold" data-extlinktrack="supportingcontent_external" data-linktrack="supportingcontent_internal" href="{{ related_link.getLink() }}">{{ related_link.getTitle() }}</a>
```
Comment (Tasiobg) [inline src/DsAmen/Presenters/Domain/RelatedLink/related_link.html.twig]: I think you want:
programmes_relatedlink_internal
programmes_relatedlink_external

**Comment by votemike** [comment]
screen shot 2017-12-20 at 11 49 42
screen shot 2017-12-20 at 11 47 44

**Pull request merged**